pkg/reconciler/mongodb/mongodbsource.go

```diff
@@ -20,23 +20,37 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"net/url"
 
 	"go.uber.org/zap"
 	"knative.dev/pkg/logging"
 
 	v1alpha1 "github.com/googleinterns/knative-source-mongodb/pkg/apis/sources/v1alpha1"
 	mongodbsource "github.com/googleinterns/knative-source-mongodb/pkg/client/injection/reconciler/sources/v1alpha1/mongodbsource"
+	"github.com/googleinterns/knative-source-mongodb/pkg/reconciler/mongodb/resources"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
+	appsv1 "k8s.io/api/apps/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	appsv1listers "k8s.io/client-go/listers/apps/v1"
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	"knative.dev/pkg/apis"
 	reconciler "knative.dev/pkg/reconciler"
 	"knative.dev/pkg/resolver"
 )
 
+// Declare Constants
+const (
+	// raImageEnvVar is the name of the environment variable that contains the receive adapter's
+	// image. It must be defined.
+	raImageEnvVar = "MONGODB_RA_IMAGE"
```

Comment:
I think you are just using this one from the controller.go?
then we should just define this constant there?